pyscriptjs/__mocks__/fileMock.js

```diff
@@ -1,12 +1,16 @@
 /**
- * this file mocks the `src/python/pyscript.py` file
- * since importing of `.py` files isn't usually supported
- * inside JS/TS files.
+ * this file mocks python files that are not explicitly
+ * match by a regex in jest.config.js importing of
```

Comment:
Very minor nitpicks, and they probably don't matter but would be nice to have.

`explicitly match` --> `explicitly matched`.

`jest.config.js importing of` -->. `jest.config.js. Importing of`

`is import from that` --> `is imported from that`

`module.exports = ""` --> `module.exports = "";` (missing semicolon)

Reply:
Genuinely very much appreciate these. Thank you!